### PR TITLE
updating docs for npm commands

### PR DESCRIPTION
**Pull request by chrisnojima**
@keybase/react-hackers

### CODE REVIEW

File: desktop/README.md [hunk 1]
```diff
@@ -22,3 +22,15 @@ To run:
...
+
+Available npm run commands:
+
+ - npm start: Build a development bundle and start the app off of those files (If you're running the app once)
+ - hot-server: Start the hot-reloading server
+ - start-hot: Connect to a hot-reloading server (If you're developing and want to see changes as you make them)
+
+ - build-dev: Build a development bundle
+ - build-prod: Build a production bundle
+ - start-cold: Start the app off of the build-dev bundles
```
Comment (cjb) [inline desktop/README.md]: How would you start the app from a prod bundle?  If start-cold does it, maybe s/build-dev/build-*/ ?

**Comment by cjb** [comment]
LGTM%C

File: desktop/README.md (hunk 1 above)
Comment (chrisnojima) [inline desktop/README.md]: you really have to package it cause it changes paths and things

Comment (cjb) [inline desktop/README.md]: ok cool

**Pull request merged**